openquakeplatform/openquakeplatform/vulnerability/views.py

```diff
@@ -985,9 +996,16 @@ def _stream_frag_discr_data(curr, func_obj, is_first):
         # of curves
         desc.text = 'Fragility from GVD: %s' % func_obj.name
 
+        limit_states = re.sub(
+            r"_+", "_", re.sub(r"[^a-z0-9;]", "_",
+            func_obj.fragility_func.limit_states_desc.lower())
+            ).replace(";", " ")
+        func_obj._mangled_limit_states = limit_states
+
```

Comment:
Please use a separate function for this, to be called from here and from `_stream_frag_cont_data`
And inside that function, add an explanatory comment like the one in the changelog.